autogpt/commands/image_gen.py

```diff
@@ -77,7 +77,7 @@ def generate_image_with_hf(prompt: str, filename: str) -> str:
     return f"Saved to disk:{filename}"
 
 
-def generate_image_with_dalle(prompt: str, filename: str) -> str:
+def generate_image_with_dalle(prompt: str, filename: str, size: int) -> str:
     """Generate an image with DALL-E.
 
     Args:
```

Comment:
Seems like you forgot to update the Args: within the comments regarding the new argument.